modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2968,6 +2968,32 @@ public void testUpdateObjectEntry() throws Exception {
 			values.get(_objectDefinition.getPKObjectFieldName()));
 		Assert.assertEquals(values.toString(), 22, values.size());
 
+		// Update with empty values for non-required object fields
```

Comment:
Hey @thallescezar I'd suggest you remove this comment otherwise will be asked to you write a comment for each use case above in `testUpdateObjectEntry` and sort them, feel free to decide if you prefer to use the comment pattern or not.

Thanks!